psk_fitness/psk_fitness.Client/Program.cs

```diff
@@ -10,14 +9,26 @@
 builder.Services.AddAuthorizationCore();
 builder.Services.AddCascadingAuthenticationState();
 builder.Services.AddSingleton<AuthenticationStateProvider, PersistentAuthenticationStateProvider>();
+
+
+var services = new Dictionary<Type, Type>
```

Comment:
Where is this used?

Reply:
nowhere, forgot to delete it when I was trying to do something